packages/terra-avatar/src/Avatar.scss

```diff
@@ -0,0 +1,56 @@
+@import '~terra-mixins';
+
+:local {
+  .avatar {
+    align-items: center;
+    background-color: var(--terra-avatar-background-color, rgb(222, 223, 224));
+    border: var(--terra-avatar-border, 0 none);
```

Comment:
If border is not needed as part of the default theme, you can omit the fallback style.

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/855d5fc5c14ce5dc2a29cb07d7d5575af2320962).